ui/README.md

```diff
@@ -46,13 +46,14 @@ actionHandler.registerEvent('yourExtension.onDidSaveTextDocument', vscode.worksp
 
 ### Display Azure Resources
 Follow these steps to create your basic Azure Tree:
-1. Implement an `IAzureTreeItem` (or `IAzureParentTreeItem`) describing the items to be displayed under your subscription:
+1. Implement an `AzureTreeItem` (or `AzureParentTreeItem`) describing the items to be displayed under your subscription:
     ```typescript
-    export class WebAppTreeItem implements IAzureTreeItem {
+    export class WebAppTreeItem extends AzureTreeItem {
         public static contextValue: string = 'azureWebApp';
         public readonly contextValue: string = WebAppTreeItem.contextValue;
         private readonly _site: Site;
-        constructor(site: Site) {
+        constructor(parent: AzureParentTreeItem, site: Site) {
+            super(parent);
             this._site = site;
         }
 
```

Comment:
Is `get id()` and `get label()` still accurate for the readme?

Reply:
Yes